kube/overlays/stable-with-resource-limits/set-resource-limits.yaml

```
@@ -67,3 +67,31 @@ spec:
             limits:
               cpu: 1
               memory: 512Mi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-minio
+spec:
+  template:
+    spec:
+      containers:
+        - name: airbyte-minio
+          resources:
+            limits:
+              cpu: 1
+              memory: 512Mi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-pod-sweeper
+spec:
+  template:
+    spec:
+      containers:
+        - name: airbyte-pod-sweeper
+          resources:
+            limits:
+              cpu: 1
+              memory: 512Mi
```

```suggestion
              memory: 128Mi
```

This is usually pretty lightweight